### ISSUE TITLE
"Failed to start secure grpc server open /etc/certs/cert-chain.pem" on Pilot

### ISSUE THREAD

**Issue by nixgadget** [istio/istio#10205]
**Describe the bug**
Similar to #9466 I'm observing Pilot crashing couple of times when it waits for citadel to generate the certs. It eventually recovers but would like to get to the bottom of why it doesnt start cleanly. This may or may not be related to #9466 hence raising this issue.
```
gc 1 @0.032s 4%: 0.015+2.2+1.3 ms clock, 0.031+0.33/0.28/1.8+2.7 ms cpu, 4->4->2 MB, 5 MB goal, 2 P
gc 2 @0.049s 7%: 0.011+2.8+1.8 ms clock, 0.023+0.38/0.20/2.6+3.6 ms cpu, 4->4->2 MB, 5 MB goal, 2 P
2018-11-30T05:39:20.460622Z	info	FLAG: --appNamespace=""
2018-11-30T05:39:20.460810Z	info	FLAG: --caCertFile="/etc/istio/certs/root-cert.pem"
2018-11-30T05:39:20.460826Z	info	FLAG: --certFile="/etc/istio/certs/cert-chain.pem"
2018-11-30T05:39:20.460934Z	info	FLAG: --clusterRegistriesNamespace=""
2018-11-30T05:39:20.460951Z	info	FLAG: --configDir=""
2018-11-30T05:39:20.461019Z	info	FLAG: --consulserverInterval="2s"
2018-11-30T05:39:20.461031Z	info	FLAG: --consulserverURL=""
2018-11-30T05:39:20.461067Z	info	FLAG: --ctrlz_address="127.0.0.1"
2018-11-30T05:39:20.461134Z	info	FLAG: --ctrlz_port="9876"
2018-11-30T05:39:20.461148Z	info	FLAG: --disable-install-crds="false"
2018-11-30T05:39:20.461207Z	info	FLAG: --discoveryCache="true"
2018-11-30T05:39:20.461218Z	info	FLAG: --domain="cluster.local"
2018-11-30T05:39:20.461223Z	info	FLAG: --grpcAddr=":15010"
2018-11-30T05:39:20.461303Z	info	FLAG: --help="false"
2018-11-30T05:39:20.461314Z	info	FLAG: --httpAddr=":8080"
2018-11-30T05:39:20.461319Z	info	FLAG: --keyFile="/etc/istio/certs/key.pem"
2018-11-30T05:39:20.461380Z	info	FLAG: --kubeconfig=""
2018-11-30T05:39:20.461384Z	info	FLAG: --log_as_json="false"
2018-11-30T05:39:20.461457Z	info	FLAG: --log_caller=""
2018-11-30T05:39:20.461468Z	info	FLAG: --log_output_level="default:info"
2018-11-30T05:39:20.461505Z	info	FLAG: --log_rotate=""
2018-11-30T05:39:20.461564Z	info	FLAG: --log_rotate_max_age="30"
2018-11-30T05:39:20.461575Z	info	FLAG: --log_rotate_max_backups="1000"
2018-11-30T05:39:20.461635Z	info	FLAG: --log_rotate_max_size="104857600"
2018-11-30T05:39:20.461646Z	info	FLAG: --log_stacktrace_level="default:none"
2018-11-30T05:39:20.461688Z	info	FLAG: --log_target="[stdout]"
2018-11-30T05:39:20.461749Z	info	FLAG: --mcpMaxMsgSize="4194304"
2018-11-30T05:39:20.461826Z	info	FLAG: --mcpServerAddrs="[]"
2018-11-30T05:39:20.461838Z	info	FLAG: --meshConfig="/etc/istio/config/mesh"
2018-11-30T05:39:20.461914Z	info	FLAG: --monitoringAddr=":9093"
2018-11-30T05:39:20.461980Z	info	FLAG: --namespace=""
2018-11-30T05:39:20.461991Z	info	FLAG: --networksConfig="/etc/istio/config/meshNetworks"
2018-11-30T05:39:20.462034Z	info	FLAG: --plugins="[authn,authz,health,mixer,envoyfilter]"
2018-11-30T05:39:20.462095Z	info	FLAG: --profile="true"
2018-11-30T05:39:20.462111Z	info	FLAG: --registries="[Kubernetes]"
2018-11-30T05:39:20.462172Z	info	FLAG: --resync="1m0s"
2018-11-30T05:39:20.462210Z	info	FLAG: --secureGrpcAddr=":15012"
2018-11-30T05:39:20.466921Z	info	mesh configuration (*v1alpha1.MeshConfig)(0xc4202665a0)(mixer_check_server:"istio-policy.control.svc.cluster.local:15004" mixer_report_server:"istio-telemetry.control.svc.cluster.local:15004" disable_policy_checks:true proxy_listen_port:15001 connect_timeout:<seconds:1 > ingress_class:"istio" ingress_controller_mode:STRICT enable_tracing:true access_log_file:"/dev/stdout" default_config:<config_path:"/etc/istio/proxy" binary_path:"/usr/local/bin/envoy" service_cluster:"istio-proxy" drain_duration:<seconds:45 > parent_shutdown_duration:<seconds:60 > discovery_address:"istio-pilot.control:15011" connect_timeout:<seconds:10 > proxy_admin_port:15000 control_plane_auth_policy:MUTUAL_TLS stat_name_length:189 tracing:<zipkin:<address:"jg-jaeger.monitoring.svc.cluster.local:9411" > > > )

2018-11-30T05:39:20.466953Z	info	version root@e34a3c8d4731-docker.io/istio-1.1.0-snapshot.3-e3653d5a8244c2f30e2e0600cb5af956a575bae1-dirty-Modified
2018-11-30T05:39:20.467483Z	info	flags (*bootstrap.PilotArgs)(0xc420414380)({
 DiscoveryOptions: (envoy.DiscoveryServiceOptions) {
  HTTPAddr: (string) (len=5) ":8080",
  GrpcAddr: (string) (len=6) ":15010",
  SecureGrpcAddr: (string) (len=6) ":15012",
  MonitoringAddr: (string) (len=5) ":9093",
  EnableProfiling: (bool) true,
  EnableCaching: (bool) true
 },
 Namespace: (string) (len=7) "control",
 Mesh: (bootstrap.MeshArgs) {
  ConfigFile: (string) (len=22) "/etc/istio/config/mesh",
  MixerAddress: (string) "",
  RdsRefreshDelay: (*types.Duration)(<nil>)
 },
 Config: (bootstrap.ConfigArgs) {
  ClusterRegistriesNamespace: (string) (len=7) "control",
  KubeConfig: (string) "",
  ControllerOptions: (kube.ControllerOptions) {
   WatchedNamespace: (string) "",
   ResyncPeriod: (time.Duration) 1m0s,
   DomainSuffix: (string) (len=13) "cluster.local",
   ClusterID: (string) "",
   XDSUpdater: (model.XDSUpdater) <nil>,
   stop: (chan struct {}) <nil>
  },
  FileDir: (string) "",
  DisableInstallCRDs: (bool) false,
  Controller: (model.ConfigStoreCache) <nil>
 },
 Service: (bootstrap.ServiceArgs) {
  Registries: ([]string) (len=1 cap=1) {
   (string) (len=10) "Kubernetes"
  },
  Consul: (bootstrap.ConsulArgs) {
   Config: (string) "",
   ServerURL: (string) "",
   Interval: (time.Duration) 2s
  }
 },
 MeshConfig: (*v1alpha1.MeshConfig)(<nil>),
 NetworksConfigFile: (string) (len=30) "/etc/istio/config/meshNetworks",
 CtrlZOptions: (*ctrlz.Options)(0xc4205512c0)({
  Port: (uint16) 9876,
  Address: (string) (len=9) "127.0.0.1"
 }),
 Plugins: ([]string) (len=5 cap=5) {
  (string) (len=5) "authn",
  (string) (len=5) "authz",
  (string) (len=6) "health",
  (string) (len=5) "mixer",
  (string) (len=11) "envoyfilter"
 },
 MCPServerAddrs: ([]string) {
 },
 MCPCredentialOptions: (*creds.Options)(0xc420567c20)({
  CertificateFile: (string) (len=31) "/etc/istio/certs/cert-chain.pem",
  KeyFile: (string) (len=24) "/etc/istio/certs/key.pem",
  CACertificateFile: (string) (len=30) "/etc/istio/certs/root-cert.pem"
 }),
 MCPMaxMessageSize: (int) 4194304
})

2018-11-30T05:39:20.468031Z	info	mesh networks configuration (*v1alpha1.MeshNetworks)(0xc42000c6c8)()

2018-11-30T05:39:20.469982Z	info	registering for apiVersion authentication.istio.io/v1alpha1
gc 3 @0.077s 10%: 0.009+3.1+4.4 ms clock, 0.019+0.24/1.1/2.8+8.9 ms cpu, 5->5->3 MB, 6 MB goal, 2 P
2018-11-30T05:39:20.487898Z	info	registering for apiVersion rbac.istio.io/v1alpha1
2018-11-30T05:39:20.498523Z	info	registering for apiVersion networking.istio.io/v1alpha3
2018-11-30T05:39:20.510761Z	info	registering for apiVersion config.istio.io/v1alpha2
2018-11-30T05:39:20.519277Z	info	CRD controller watching namespaces ""
2018-11-30T05:39:20.519743Z	info	Ingress controller watching namespaces ""
2018-11-30T05:39:20.531004Z	info	Adding Kubernetes registry adapter
2018-11-30T05:39:20.531035Z	info	Primary Cluster name: Kubernetes
2018-11-30T05:39:20.531045Z	info	Service controller watching namespace "" for services, endpoints, nodes and pods, refresh 1m0s
2018-11-30T05:39:20.531462Z	info	ads	Starting ADS server with rateLimiter=10 burst=100
2018-11-30T05:39:20.531698Z	info	Setting up event handlers
2018-11-30T05:39:20.531798Z	info	Starting Secrets controller
2018-11-30T05:39:20.531907Z	info	Waiting for informer caches to sync
2018-11-30T05:39:20.536957Z	info	Discovery service started at http=[::]:8080 grpc=[::]:15010 secure grpc=[::]:15012
2018-11-30T05:39:20.537516Z	info	ControlZ available at 10.12.1.88:9876
2018-11-30T05:39:20.538510Z	info	attempting to acquire leader lease  control/istio-ingress-controller-leader-istio...
2018-11-30T05:39:20.539373Z	error	Failed to start secure grpc server open /etc/certs/cert-chain.pem: no such file or directory
```

**Expected behavior**
Pilot to start without crashing

**Steps to reproduce the bug**
Just following the stock helm deployment would lead to this problem

**Version**
**Istio**: Istio 1.1.0-snapshot.3
**GKE**: 1.11.2-gke.18

**Installation**
Stock helm deployment with mTLS and controlPlaneSecurityEnabled.

**Environment**
GKE

**Comment by nixgadget** [comment]
@hzxuzhonghu Yeah when Pilot crashes `istio-pilot-service-account` is missing. Heres the citadel log. It does take several seconds to create the secret.

```
Flag --grpc-hostname has been deprecated, please use --grpc-host-identities instead
2018-11-30T10:56:17.105010Z	warn	Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
2018-11-30T10:56:17.105635Z	info	ControlZ available at 10.12.1.99:9876
2018-11-30T10:56:17.106241Z	info	Use self-signed certificate as the CA certificate
2018-11-30T10:56:17.239759Z	info	adding registry entry "k8s.cluster.local" -> "k8s.cluster.local"
2018-11-30T10:56:17.240747Z	info	Citadel monitor has started.
2018-11-30T10:56:17.240770Z	info	Citadel has started
2018-11-30T10:56:17.240807Z	info	Monitor server started.
2018-11-30T10:56:17.240793Z	info	Starting GRPC server on port 8060
2018-11-30T10:56:17.248919Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-sidecar-injector-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-sidecar-injector-service-account"
2018-11-30T10:56:17.248981Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/clusterrole-aggregation-controller" -> "spiffe://cluster.local/ns/kube-system/sa/clusterrole-aggregation-controller"
2018-11-30T10:56:17.249086Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/fluentd-gcp-scaler" -> "spiffe://cluster.local/ns/kube-system/sa/fluentd-gcp-scaler"
2018-11-30T10:56:17.249093Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/kube-dns-autoscaler" -> "spiffe://cluster.local/ns/kube-system/sa/kube-dns-autoscaler"
2018-11-30T10:56:17.249102Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/default" -> "spiffe://cluster.local/ns/control/sa/default"
2018-11-30T10:56:17.249108Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/disruption-controller" -> "spiffe://cluster.local/ns/kube-system/sa/disruption-controller"
2018-11-30T10:56:17.249114Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/pv-protection-controller" -> "spiffe://cluster.local/ns/kube-system/sa/pv-protection-controller"
2018-11-30T10:56:17.249121Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/resourcequota-controller" -> "spiffe://cluster.local/ns/kube-system/sa/resourcequota-controller"
2018-11-30T10:56:17.249246Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/daemon-set-controller" -> "spiffe://cluster.local/ns/kube-system/sa/daemon-set-controller"
2018-11-30T10:56:17.249255Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/service-controller" -> "spiffe://cluster.local/ns/kube-system/sa/service-controller"
2018-11-30T10:56:17.249265Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/generic-garbage-collector" -> "spiffe://cluster.local/ns/kube-system/sa/generic-garbage-collector"
2018-11-30T10:56:17.249272Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/route-controller" -> "spiffe://cluster.local/ns/kube-system/sa/route-controller"
2018-11-30T10:56:17.249287Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/attachdetach-controller" -> "spiffe://cluster.local/ns/kube-system/sa/attachdetach-controller"
2018-11-30T10:56:17.249296Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/replication-controller" -> "spiffe://cluster.local/ns/kube-system/sa/replication-controller"
2018-11-30T10:56:17.249302Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/tiller" -> "spiffe://cluster.local/ns/kube-system/sa/tiller"
2018-11-30T10:56:17.249308Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-mixer-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-mixer-service-account"
2018-11-30T10:56:17.250281Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/event-exporter-sa" -> "spiffe://cluster.local/ns/kube-system/sa/event-exporter-sa"
2018-11-30T10:56:17.250308Z	info	adding registry entry "spiffe://cluster.local/ns/istio-system/sa/default" -> "spiffe://cluster.local/ns/istio-system/sa/default"
2018-11-30T10:56:17.250317Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-egressgateway-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-egressgateway-service-account"
2018-11-30T10:56:17.250324Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-pilot-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-pilot-service-account"
2018-11-30T10:56:17.250331Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/replicaset-controller" -> "spiffe://cluster.local/ns/kube-system/sa/replicaset-controller"
2018-11-30T10:56:17.250338Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/deployment-controller" -> "spiffe://cluster.local/ns/kube-system/sa/deployment-controller"
2018-11-30T10:56:17.250346Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/certificate-controller" -> "spiffe://cluster.local/ns/kube-system/sa/certificate-controller"
2018-11-30T10:56:17.250354Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/default" -> "spiffe://cluster.local/ns/kube-system/sa/default"
2018-11-30T10:56:17.250360Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-citadel-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-citadel-service-account"
2018-11-30T10:56:17.250368Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-galley-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-galley-service-account"
2018-11-30T10:56:17.250382Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-ingressgateway-service-account" -> "spiffe://cluster.local/ns/control/sa/istio-ingressgateway-service-account"
2018-11-30T10:56:17.250407Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/cloud-provider" -> "spiffe://cluster.local/ns/kube-system/sa/cloud-provider"
2018-11-30T10:56:17.250415Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/endpoint-controller" -> "spiffe://cluster.local/ns/kube-system/sa/endpoint-controller"
2018-11-30T10:56:17.250422Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/statefulset-controller" -> "spiffe://cluster.local/ns/kube-system/sa/statefulset-controller"
2018-11-30T10:56:17.250427Z	info	adding registry entry "spiffe://cluster.local/ns/kube-public/sa/default" -> "spiffe://cluster.local/ns/kube-public/sa/default"
2018-11-30T10:56:17.250434Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/cronjob-controller" -> "spiffe://cluster.local/ns/kube-system/sa/cronjob-controller"
2018-11-30T10:56:17.250440Z	info	adding registry entry "spiffe://cluster.local/ns/default/sa/default" -> "spiffe://cluster.local/ns/default/sa/default"
2018-11-30T10:56:17.250447Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/fluentd-gcp" -> "spiffe://cluster.local/ns/kube-system/sa/fluentd-gcp"
2018-11-30T10:56:17.250452Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/metrics-server" -> "spiffe://cluster.local/ns/kube-system/sa/metrics-server"
2018-11-30T10:56:17.250458Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/heapster" -> "spiffe://cluster.local/ns/kube-system/sa/heapster"
2018-11-30T10:56:17.250472Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/pvc-protection-controller" -> "spiffe://cluster.local/ns/kube-system/sa/pvc-protection-controller"
2018-11-30T10:56:17.250880Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/persistent-volume-binder" -> "spiffe://cluster.local/ns/kube-system/sa/persistent-volume-binder"
2018-11-30T10:56:17.250890Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/horizontal-pod-autoscaler" -> "spiffe://cluster.local/ns/kube-system/sa/horizontal-pod-autoscaler"
2018-11-30T10:56:17.250897Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/namespace-controller" -> "spiffe://cluster.local/ns/kube-system/sa/namespace-controller"
2018-11-30T10:56:17.250905Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/kube-dns" -> "spiffe://cluster.local/ns/kube-system/sa/kube-dns"
2018-11-30T10:56:17.250912Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/ttl-controller" -> "spiffe://cluster.local/ns/kube-system/sa/ttl-controller"
2018-11-30T10:56:17.250919Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/expand-controller" -> "spiffe://cluster.local/ns/kube-system/sa/expand-controller"
2018-11-30T10:56:17.250926Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/pod-garbage-collector" -> "spiffe://cluster.local/ns/kube-system/sa/pod-garbage-collector"
2018-11-30T10:56:17.250935Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/job-controller" -> "spiffe://cluster.local/ns/kube-system/sa/job-controller"
2018-11-30T10:56:17.250941Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/node-controller" -> "spiffe://cluster.local/ns/kube-system/sa/node-controller"
2018-11-30T10:56:17.250948Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/service-account-controller" -> "spiffe://cluster.local/ns/kube-system/sa/service-account-controller"
2018-11-30T10:56:17.250969Z	info	adding registry entry "spiffe://cluster.local/ns/kube-system/sa/metadata-proxy" -> "spiffe://cluster.local/ns/kube-system/sa/metadata-proxy"
2018-11-30T10:56:17.250976Z	info	adding registry entry "spiffe://cluster.local/ns/control/sa/istio-citadel-custom-job-cr-sa" -> "spiffe://cluster.local/ns/control/sa/istio-citadel-custom-job-cr-sa"
2018-11-30T10:56:17.491238Z	info	Istio secret for service account "cronjob-controller" in namespace "kube-system" has been created
2018-11-30T10:56:18.378781Z	info	Istio secret for service account "default" in namespace "default" has been created
2018-11-30T10:56:19.576020Z	info	Istio secret for service account "fluentd-gcp" in namespace "kube-system" has been created
2018-11-30T10:56:20.284217Z	info	Istio secret for service account "metrics-server" in namespace "kube-system" has been created
2018-11-30T10:56:20.767366Z	info	Istio secret for service account "pvc-protection-controller" in namespace "kube-system" has been created
2018-11-30T10:56:20.997239Z	info	Istio secret for service account "persistent-volume-binder" in namespace "kube-system" has been created
2018-11-30T10:56:21.518024Z	info	Istio secret for service account "horizontal-pod-autoscaler" in namespace "kube-system" has been created
2018-11-30T10:56:21.702348Z	info	Istio secret for service account "namespace-controller" in namespace "kube-system" has been created
2018-11-30T10:56:22.226978Z	info	Istio secret for service account "heapster" in namespace "kube-system" has been created
2018-11-30T10:56:22.422040Z	info	Istio secret for service account "ttl-controller" in namespace "kube-system" has been created
2018-11-30T10:56:22.863129Z	info	Istio secret for service account "expand-controller" in namespace "kube-system" has been created
2018-11-30T10:56:23.152721Z	info	Istio secret for service account "pod-garbage-collector" in namespace "kube-system" has been created
2018-11-30T10:56:23.616226Z	info	Istio secret for service account "job-controller" in namespace "kube-system" has been created
2018-11-30T10:56:23.933562Z	info	Istio secret for service account "kube-dns" in namespace "kube-system" has been created
2018-11-30T10:56:24.271055Z	info	Istio secret for service account "node-controller" in namespace "kube-system" has been created
2018-11-30T10:56:24.514182Z	info	Istio secret for service account "service-account-controller" in namespace "kube-system" has been created
2018-11-30T10:56:25.093812Z	info	Istio secret for service account "metadata-proxy" in namespace "kube-system" has been created
2018-11-30T10:56:25.666029Z	info	Istio secret for service account "istio-citadel-custom-job-cr-sa" in namespace "control" has been created
2018-11-30T10:56:25.989844Z	info	Istio secret for service account "clusterrole-aggregation-controller" in namespace "kube-system" has been created
2018-11-30T10:56:26.826592Z	info	Istio secret for service account "fluentd-gcp-scaler" in namespace "kube-system" has been created
2018-11-30T10:56:28.045177Z	info	Istio secret for service account "kube-dns-autoscaler" in namespace "kube-system" has been created
2018-11-30T10:56:28.564097Z	info	Istio secret for service account "default" in namespace "control" has been created
2018-11-30T10:56:28.770386Z	info	Istio secret for service account "istio-sidecar-injector-service-account" in namespace "control" has been created
2018-11-30T10:56:29.150996Z	info	Istio secret for service account "daemon-set-controller" in namespace "kube-system" has been created
2018-11-30T10:56:29.362312Z	info	Istio secret for service account "service-controller" in namespace "kube-system" has been created
2018-11-30T10:56:29.681120Z	info	Istio secret for service account "generic-garbage-collector" in namespace "kube-system" has been created
2018-11-30T10:56:29.994672Z	info	Istio secret for service account "route-controller" in namespace "kube-system" has been created
2018-11-30T10:56:30.133314Z	info	Istio secret for service account "attachdetach-controller" in namespace "kube-system" has been created
2018-11-30T10:56:30.308043Z	info	Istio secret for service account "disruption-controller" in namespace "kube-system" has been created
2018-11-30T10:56:30.765612Z	info	Istio secret for service account "pv-protection-controller" in namespace "kube-system" has been created
2018-11-30T10:56:30.999440Z	info	Istio secret for service account "resourcequota-controller" in namespace "kube-system" has been created
2018-11-30T10:56:31.486526Z	info	Istio secret for service account "replication-controller" in namespace "kube-system" has been created
2018-11-30T10:56:31.921497Z	info	Istio secret for service account "tiller" in namespace "kube-system" has been created
2018-11-30T10:56:32.355379Z	info	Istio secret for service account "istio-mixer-service-account" in namespace "control" has been created
2018-11-30T10:56:32.713491Z	info	Istio secret for service account "istio-pilot-service-account" in namespace "control" has been created
```

**Comment by elevran** [comment]
Seems to be same root cause as #9466 (certs not ready).
Need to
- generate certificates faster (may be possible with SDS and nodeagent); or
- wait (e.g., using init container) for secrets to be populated
